src/utils/beforeAll.js

```diff
@@ -83,11 +83,10 @@ function checkRequiredModules({ allModules, required }) {
 * @param {Object} options.allModules - All active modules
 * @returns {Promise<Object[]>} All modules configurations
 */
-async function getModulesConfiguration({ allModules, directories }) {
+function getModulesConfiguration({ allModules, directories }) {
     const configDir = directories.config || DIRECTORIES.config;
-    const config = await Promise.all(getConfigs({ modules: allModules, suffix: `${configDir}/index.js` }));
 
-    return config;
+    return Promise.all(getConfigs({ modules: allModules, suffix: `${configDir}/index.js` }));
```

Comment:
It's redundant - Code Quality tools are telling that there is no need of creating variable and returning it - memory allocation.

Also it's not true. For example function checkRequiredModules is returning Promise instead of awaiting for result